src/agent/tests/CMakeLists.txt

```diff
@@ -37,3 +37,9 @@ configure_target(cmd_ln_parser_test)
 target_include_directories(cmd_ln_parser_test PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../include)
 target_link_libraries(cmd_ln_parser_test PRIVATE GTest::gtest)
 add_test(NAME CommandlineParserTest COMMAND cmd_ln_parser_test)
+
+add_executable(unix_daemon_test unix_daemon_test.cpp)
+configure_target(unix_daemon_test)
+target_include_directories(unix_daemon_test PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/../include ${CMAKE_CURRENT_SOURCE_DIR}/../src)
+target_link_libraries(unix_daemon_test PRIVATE Agent GTest::gtest)
+add_test(NAME UnixDaemonTest COMMAND unix_daemon_test)
```

Comment:
It looks like there's a new line missing at the end of the file. Could you add one, please?

Reply:
Fixed